src/test/java/HomeWork/Inputs.java

```diff
@@ -0,0 +1,23 @@
+package HomeWork;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Inputs extends PreCondition {
+    @Test
+    public void inputs() {
+        driver.navigate().to("https://the-internet.herokuapp.com/inputs");
+        Assert.assertTrue(driver.findElements(By.cssSelector("input[type=number]")).get(0).getAttribute("value").equals(""));
+        driver.findElement(By.cssSelector("input[type=number]")).sendKeys("zxcvb%^&");
+        Assert.assertTrue(driver.findElements(By.cssSelector("input[type=number]")).get(0).getAttribute("value").equals(""));
+        driver.findElement(By.cssSelector("input[type=number]")).sendKeys("122");
+        Assert.assertTrue(driver.findElements(By.cssSelector("input[type=number]")).get(0).getAttribute("value").equals("122"));
+        driver.findElement(By.cssSelector("input[type=number]")).sendKeys(Keys.ARROW_UP);
```

Comment:
It makes no sense to find input element each time. You can find it once at the very beginning and then use it for all further actions

Reply:
like WebElemment input = driver.findElement(....)
input.sendKeys();
... 
input.sendKeys()
input.getAttribute("value")